robosuite/controllers/composite/composite_controller_factory.py

```diff
@@ -18,6 +19,57 @@ def validate_composite_controller_config(config: dict):
             raise ValueError
 
 
+def is_stale_controller_config(config: Dict):
+    """
+    Checks if a controller config is in the format from robosuite versions <= 1.4.1.
```

Comment:
I guess this function is now also being used to check if a config is a part-only controller (which we use post v1.4.1 in `demo_control.py`) versus a full composite controller. Maybe we can call it `is_part_controller_config` instead and update the doc string accordingly?

Reply:
Specifically checks for arm controllers, will rename and update accordingly

Reply:
I see: I think it's possible to use IK/OSC (and, of course, joint pos/vel/torque) for non-arm controllers